src/main.js

```diff
@@ -109,11 +110,11 @@ function saveCover() {
   }
 }
 function displaySavedCovers() {
-  viewSavedCovers() 
+  viewSavedCovers();
   viewSavedCoverLocation.innerHTML = '';
   for (var i = 0; i < savedCovers.length; i++) {
-    viewSavedCoverLocation.innerHTML +=
-    `<section class = "main-cover">
+    viewSavedCoverLocation.innerHTML += 
+    `<section class = "mini-cover">
```

Comment:
html convention is to run attributes and values together. I.e. `class="something"`, as opposed to `class = "something"`